src/Symfony/Component/HttpKernel/EventListener/SessionListener.php

```diff
@@ -25,6 +25,8 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
+ * @internal
```

Comment:
`since Symfony 5.3`? but why?

Reply:
if not, we have to deal with deprecations messages telling the user to stop injecting `session`, but to inject `session_factory` instead. I don't think this class worth it

Reply:
side note, this class extends `AbstractSessionListener` which is `internal`. And I think in 6.0 we could refactor these classes and merge theme.

Reply:
We could merge the classes on 5.4 already, if we wanted. 🤔

Reply:
Do you think it's a good idea and I should do it in this PR?

Reply:
sounds good for another PR